staging/src/k8s.io/kubectl/pkg/cmd/create/create_job_test.go

```diff
@@ -137,9 +137,9 @@ func TestCreateJob(t *testing.T) {
 
 func TestCreateJobFromCronJob(t *testing.T) {
```

Comment:
exercise both the v1 and v1beta1 variants?

Reply:
Done.